mesa/visualization/ModularVisualization.py

```diff
@@ -250,34 +250,60 @@ def on_message(self, message):
 class ModularServer(tornado.web.Application):
     """Main visualization application."""
 
-    verbose = True
+    EXCLUDE_LIST = ("width", "height")
 
-    port = int(os.getenv("PORT", 8521))  # Default port to listen on
-    max_steps = 100000
```

Comment:
What is the rational of this attribute? I haven't found any application. Why is it set to 100000? At least, there should be a comment, I guess.

Reply:
Regardless of the situation with this attribute, it is outside the scope of this PR. If you mix it in this PR, it will be hard to track the changes.

Reply:
Hm,I don't understand. In which scope would it be if not "refactor ModularServer"?

Reply:
The scope is to move the class attributes to be inside `__init__`. The term "refactor" is too broad and may include lots of things. Anyway, let's keep the PR small, so it can be quickly merged.